src/features/accounts/components/social-login/social-login.tsx

```diff
@@ -135,20 +129,33 @@ export function SocialLogin({ onSuccess }: { onSuccess: () => void }) {
           {isConnecting ? <Spinner size="sm" /> : null}
         </HStack>
         <HStack spacing={6}>
-          {socials.map(social => {
-            return (
-              <IconButton
-                key={social.provider}
-                aria-label={social.provider}
-                icon={<social.Icon />}
```

Comment:
This section is the only logical change in this pull request, the rest is all renaming imports. For some reason, `<social.Icon />` wasn't working anymore after changing the paths to the shared component folder. I'll take a look at this again once we've got everything pulled into a separate project.